mudlet-mapper.xml

```diff
@@ -4835,7 +4835,7 @@ mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
 
 
-local newversion = &quot;developer&quot;
+local newversion = &quot;18.3.1&quot;
```

Comment:
Could you undo this change? The release script relies on the version being `developer`.